lusidtools/lpt/txn_config_yaml.py

```diff
@@ -387,10 +388,11 @@ def alias_con(loader, node):
         return (
             s[0],
             s[1],
+            s[4],
             s[3],
             s[2],
-            unabbrev(s[4]),
-            (len(s) > 5 and s[5] == "default"),
+            unabbrev(s[5]),
+            (len(s) > 6 and s[6] == "default"),
```

Comment:
Would this going to break any existing back-up versions of txn_configs when trying to set configurations? or any jobs that might currently configure TXN types from a YAML file?

Reply:
In theory, yes. In exactly the same way that lpt was broken because of an sdk change, this change could cause issues for lpt users. Good point. Let's discuss

Reply:
agreed